ee/api/test/test_event_definition.py

```diff
@@ -242,3 +243,99 @@ def test_event_definition_no_duplicate_tags(self):
         )
 
         self.assertListEqual(sorted(response.json()["tags"]), ["a", "b"])
+
+    def test_included_ids_filter(self):
+        super(LicenseManager, cast(LicenseManager, License.objects)).create(
+            plan="enterprise", valid_until=timezone.datetime(2500, 1, 19, 3, 14, 7)
+        )
+        # rated_app, installed_app
+        rated_app_event = EnterpriseEventDefinition.objects.create(team=self.team, name="rated_app")
+        installed_app_event = EnterpriseEventDefinition.objects.create(team=self.team, name="installed_app")
+        ids = [rated_app_event.id, installed_app_event.id]
+
+        response = self.client.get("/api/projects/@current/event_definitions/?search=app")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.json()["count"], 2)  # installed_app, rated_app
+        self.assertEqual(response.json()["results"][0]["name"], "installed_app")
+        self.assertEqual(response.json()["results"][1]["name"], "rated_app")
+
+        included_ids_str = f'["{str(ids[0])}"]'
```

Comment:
q: why is this a string that starts with braces (a.k.a. a string that looks like an array) ?

Reply:
Kind of ugly but I figured it doesn't really matter since this lies in a test. I need to pass in a jsonified array that is later url encoded. For some reason `urlencode()` will escape the quotes in the array if the array object is passed into the method, which `json.load()` can't parse.